x-pack/plugins/ml/public/application/components/annotations/annotations_table/annotations_table.js

```diff
@@ -447,38 +482,116 @@ export class AnnotationsTable extends Component {
       });
     }
 
-    columns.push({
-      align: RIGHT_ALIGNMENT,
-      width: '60px',
-      name: i18n.translate('xpack.ml.annotationsTable.actionsColumnName', {
-        defaultMessage: 'Actions',
-      }),
-      actions,
-    });
-
     const getRowProps = (item) => {
       return {
         onMouseOver: () => this.onMouseOverRow(item),
         onMouseLeave: () => this.onMouseLeaveRow(),
       };
     };
+    let filterOptions = [];
+    if (this.props.aggregations?.event?.buckets) {
+      const buckets = this.props.aggregations.event.buckets;
+      const foundUser = buckets.findIndex((d) => d.key === ANNOTATION_EVENT_USER) > -1;
+      filterOptions = foundUser
+        ? buckets
+        : [{ key: ANNOTATION_EVENT_USER, doc_count: 0 }, ...buckets];
+    }
+    const filters = [
+      {
+        type: 'field_value_selection',
+        field: 'event',
+        name: 'Event',
+        multiSelect: 'or',
+        options: filterOptions.map((field) => ({
+          value: field.key,
+          name: field.key,
+          view: `${field.key} (${field.doc_count})`,
+        })),
+      },
+    ];
+
+    if (
```

Comment:
We should show the partitioning field columns in the table in the Anomaly Explorer too. In fact they are even more important in this view, as it is showing anomalies across all partitions.

Reply:
I updated to include all 3 columns in AE and SMV 
<img width="1461" alt="Screen Shot 2020-07-08 at 5 10 43 PM" src="https://user-images.githubusercontent.com/43350163/86975481-07ba3b80-c13e-11ea-8afb-b591275ae179.png">

I can also add the detector column to AE as well if we also want that. I omitted it for now because of the space constraint.

Reply:
Agree not to display the detector column in the table due to lack of space. Would be nice to only show the `partition` , `over` and `by` fields according to the detectors of the jobs being viewed, but this can always be done in a follow-up.

Reply:
For a job with 1 detector with no partitioning fields, I see all three columns `partition`, `by` and `over` show up here:

![image](https://user-images.githubusercontent.com/7405507/87329798-6f51fb80-c52f-11ea-8edc-aa6e65d10b81.png)

Reply:
Tested and working as expected with https://github.com/elastic/kibana/pull/70198/commits/487178366e43d8ebe48bed9e1acb13a7efb8da25.